mtp_api/apps/security/serializers.py

```diff
@@ -218,7 +217,6 @@ def update(self, instance, validated_data):
 
 
 class CheckSerializer(serializers.ModelSerializer):
-    credit = SecurityCreditSerializer()
```

Comment:
this line is still required as this is used unsubclassed in `security.views.CheckView`

Reply:
wait no it doesn't anymore. what's the distinction between `CheckSerializer` and `CheckCreditSerializer`?

Reply:
`CheckCreditSerializer` serializes the attached credit, `CheckSerializer` does not. There are other ways to do this other than subclassing the serializer though, I can point you in the direction of a package that has this as one of it's core usecases. I didn't think it was worth adding just for this, but i'm happy to do so

Reply:
oh i don't think we need any extra package just for this. i'm completely fine with serialiser subclassing!

at first i thought `CheckSerializer` was still being used in the check view, now it looks like it's never used directly except as the superclass of `CheckCreditSerializer`